src/schnetpack/configs/task/scheduler/multistep.yaml

```diff
@@ -0,0 +1,7 @@
+# @package task
+scheduler_cls: torch.optim.lr_scheduler.MultiStepLR
+scheduler_monitor: val_loss
+scheduler_args:
+  milestones: [3,9,15,18,24,30,36]
```

Comment:
the milestones are very specific to the experiment. I would either set them to `???` or just define the scheduler in the experiment.yaml

Reply:
I agree, done in ca4acd7070786631797886a25488d19794c7e214